software-project-management-plan.md

```diff
@@ -6,17 +6,129 @@
+| Client app UI mock-ups | Interactive user interface mock-ups representing how the U↑ mobile and desktop apps will look and respond to user interaction. |
+| System back-end and console | Server software and database for the U↑ system that can be interacted with and tested through an administrator console. |
+| Client app integration | Functional client applications that connect to the U↑ back-end and database to make and receive calls. |
+| System metrics & performance report | A report detailing the measured increases in efficiency, customer satisfaction, and system availability after implementing U↑ within the company. |
+
 ### 1.3 Evolution of SPMP
+The SPMP for the U↑ project will be under version control, so any changes will be made to the plan itself. The updated document will be made available to all project members and interested stakeholders. The most recent version will also be available in the U↑ repository.
+
+### 1.4 Schedule
+| Executive Milestones | Estimated Completion Time frame |
+| -------------------- | ------------------------------ |
+| Project plan created and approved | 2 weeks after Project Charter's initial draft is completed |
+| Visual prototypes developed | 8 weeks after requirements are finalized |
+| Software version 1 completed | 16 weeks after prototypes are completed |
+| Security and load testing of system | 4 weeks after completion of Version 1 |
+| Limited integration of system into company | 2 weeks after security and load testing |
+| Full system integration into company | 8 weeks after limited system integration |
 
 ## 2 PROJECT ORGANISATION
+The SPMP will identify the organizational entities external to the project and their interaction with the project team, as well as internal project structure and roles and responsibilities for the project. Section 2.1 describes the external interfaces to the U↑ project team, section 2.2 describes the internal structure of the team, while section 2.3 describes the roles performed by the team members.
+
+### 2.1 External Structure
+The client for this project is the TA Pengfei who is supposed to be a company wanting to use this product for internal use. All formal communication between the client and team was done through email on an as-needed basis. All team communication was done in person during class meetings or via discord as-needed. 
+
 ### 2.1 Process Model
+The U↑ project will follow an incremental and an iterative development model for its deliverables. The development will be done in several phases and each phase will represent a complete development cycle, with certain functionality of the system delivered at the end of each phase The phased approach to delivery provides flexibility in what the team will deliver, gives an opportunity to reassess the effort for each phase and allows both the team and the client to change any of the phase’s content.
+
 ### 2.2 Organisational Structure
+#### Internal Team
```

Comment:
for context, it might be good to include a brief description of the structure- just something like "the project team follows a hierarchical internal structure whose roles are listed below. the direct reports of a position are indicated by sub-bullet points."

Reply:
sounds good